### DISCUSSION TITLE
Is there a community soft-fork that allows external contributions ?

### DISCUSSION THREAD

**Discussion by iTrooz** [jgraph/drawio#4612] [Q&A]
Hey ! I love this project, but I found a few caveats with it, that make it hard to use for me as a daily usage, namely:
- being able to "import" a base diagram structure from Mermaid-like code
- live exporting to an URL (once again like Mermaid), so that I can find and edit old diagrams from my browser history, rather than having to manually save with a third party service, which I am prone to forget, given that I am not constantly logged in them

I'd love to implement theses features myself, but unfortunately, this project doesn't accept third party contributions.
I was wondering if there was a community soft-fork (or hard-fork, if its more active ?) of this project, that I could make these PRs to ?
Thanks !

**Comment by davidjgraph** [comment]
I'm not really sure why you'd need a community fork in order to that. Why not just fork the project and implement the features?

I'm not aware of any successful forks, but I haven't looked.

**Reply by iTrooz** [reply]
Hey !
The reason I was looking for a fork is that I wanted to share my improvements, so that people can use them easily. Implementing something for myself only in an obscure repo that isn't going to be seen/used by anyone has less appeal to me.
I also wanted to avoid having maintaining the fork by rebasing on upstream all the time ;)

I've seen that draw.io has a plugin system, I'll look into that, as it might be a better fit for me
